dev/build.py

```diff
@@ -40,44 +57,42 @@ def main():
     args = parse_args()
 
     # Clean up build artifacts generated by previous builds
-    for path in map(
-        Path,
-        [
-            "build",
-            "dist",
-            "mlflow.egg-info",
-            "skinny/dist",
-            "skinny/mlflow_skinny.egg_info",
-        ],
-    ):
+    paths_to_clean_up = ["build"]
+    for pkg in PACKAGES:
+        paths_to_clean_up += [
+            f"{pkg.build_path}/dist",
+            f"{pkg.build_path}/{pkg.pypi_name}.egg_info",
+        ]
+    for path in map(Path, paths_to_clean_up):
         if not path.exists():
             continue
         if path.is_file():
             path.unlink()
         else:
             shutil.rmtree(path)
 
+    package = next(p for p in PACKAGES if p.type == args.package_type)
+
     with restore_changes():
         pyproject = Path("pyproject.toml")
-        if args.package_type == "release":
+        if package.type == RELEASE:
```

Comment:
```suggestion
        if package == RELEASE:
```

do we need `.type` here?

Reply:
nope, thanks for the catch!